src/server/worker/api_server.go

```diff
@@ -471,10 +477,22 @@ func (a *APIServer) downloadData(pachClient *client.APIClient, logger *taggedLog
 }
 
 // Run user code and return the combined output of stdout and stderr.
-func (a *APIServer) runUserCode(ctx context.Context, logger *taggedLogger, environ []string, stats *pps.ProcessStats, rawDatumTimeout *types.Duration) (retErr error) {
+func (a *APIServer) runUserCode(ctx context.Context, logger *taggedLogger, environ []string, stats *pps.ProcessStats, rawDatumTimeout *types.Duration, enterpriseEnabled bool) (retErr error) {
```

Comment:
`enterpriseEnabled` is a bad name for this argument. Arguments should reflect how they'll be used, not where they came from. This should be called something like `exportMetrics`.

Reply:
I like that argument. I'm going w `exportStats` as we use the word `metrics` to denote mixpanel reporting.

Reply:
I think `metrics` or `exportMetrics` would be better actually. We use stats to denote the stats branch, and there's actually another argument to this function that references those stats. It seems really confusing to have 2 arguments to the function that reference stats when they're not the same stats.